AxonDeepSeg/morphometrics/launch_morphometrics_computation.py

```diff
@@ -192,6 +194,18 @@ def main(argv=None):
                 # Export to csv    
                 else: 
                     pd.DataFrame(x).to_csv(current_path_target.parent / filename)
+
+                # Generate the numbers image
+                number_outfile = current_path_target.parent /(str(current_path_target.with_suffix("")) +
+                                                              "_numbers_seg.png")
+                ads.imwrite(number_outfile, number_array)
+                # Generate the colored image
+                postprocessing.generate_and_save_colored_image_with_numbers(
+                    filename=current_path_target.parent /(str(current_path_target.with_suffix("")) +
+                                                          "_colored_seg.png"),
```

Comment:
Same for the suffix `_colored_seg.png`, should we add it in the [config](https://github.com/neuropoly/axondeepseg/blob/master/config.py) file?